config/samples/pods_with_pkcs11-luna.yaml

```diff
@@ -19,31 +19,44 @@ spec:
     database: BarbicanDatabasePassword
     service: BarbicanPassword
     simplecryptokek: BarbicanSimpleCryptoKEK
+    p11cryptologin: BarbicanP11Login
   preserveJobs: true
   customServiceConfig: |
     [DEFAULT]
     debug = True
+
+    [secretstore:pkcs11]
+    secret_store_plugin = store_crypto
+    crypto_plugin = p11_crypto
+
+    [p11_crypto_plugin]
+    plugin_name = PKCS11
+    library_path = /usr/local/luna/libs/64/libCryptoki2.so
+    token_serial_number = 12345
+    token_labels = some_token_label
+    slot_id = 123
+    mkek_label = some_mkek_label
+    mkek_length = 123455
+    hmac_label = some_hmac_label
+    encryption_mechanism = CKM_AES_GCM
+    aes_gcm_generate_iv = true
+    hmac_key_type = CKK_GENERIC_SECRET
+    hmac_keygen_mechanism = CKM_GENERIC_SECRET_KEY_GEN
+    hmac_keywrap_mechanism = CKM_AES_KEY_WRAP_KWP
+    key_wrap_mechanism = true
```

Comment:
This should be the name of a key wrapping mechanism, not a boolean:

`key_wrap_mechanism = CKM_AES_KEY_WRAP_WKP`

Reply:
ack.  I noticed this when I was testing, but forgot to change it here.  Good catch.

Reply:
Done